modules/utxo-lib/test/transaction_util.ts

```diff
@@ -56,24 +64,26 @@ export function parseTransactionRoundTrip<TNumber extends number | bigint, T ext
     const bigintTx = tx.clone<bigint>('bigint');
     const bigintInputs = inputs.map((input) => ({ ...input, value: BigInt(input.value) }));
     // Test UtxoPsbt.fromTransaction() implementation
-    assert.strictEqual(
-      UtxoPsbt.fromTransaction(bigintTx, bigintInputs)
-        .finalizeAllInputs()
-        .extractTransaction()
-        .toBuffer()
-        .toString('hex'),
-      buf.toString('hex')
-    );
+    if (roundTripPsbt) {
+      assert.strictEqual(
+        UtxoPsbt.fromTransaction(bigintTx, bigintInputs)
+          .finalizeAllInputs()
+          .extractTransaction()
+          .toBuffer()
+          .toString('hex'),
+        buf.toString('hex')
+      );
+
+      assert.strictEqual(
+        createPsbtFromBuffer(createPsbtFromTransaction(bigintTx, bigintInputs).toBuffer(), network)
+          .finalizeAllInputs()
+          .extractTransaction()
+          .toBuffer()
+          .toString('hex'),
+        buf.toString('hex')
+      );
+    }
     // Test UtxoPsbt.toBuffer() and UtxoPsbt.fromBuffer() implementation
```

Comment:
Should this comment move up a few lines closer to the `createPsbtFromBuffer()` call?

Reply:
oops indeed